# Shark Tank

## Due Tuesday, April 12 at 8:00 AM

_Shark Tank_ is a reality TV show. Contestants present their idea for a company to a panel of investors (a.k.a. "sharks"), who then decide whether or not to invest in that company.  The investors give a certain amount of money in exchange for a percentage stake in the company ("equity"). If you are not familiar with the show, you may want to watch at least part of one episode [here](http://abc.go.com/shows/shark-tank) to get a sense.

The data that you will examine in this lab contains the results of all contestants in the first 6 seasons of the show: 
- the name and industry of the company
- whether or not it was funded
- which sharks chose to invest (Note: There are 7 sharks, not including "Guest". They are represented by their last names.)
- if funded, the amount of money the sharks put in and the percentage equity they got in return

In [1]:
import pandas as pd
data = pd.read_csv("/data/sharktank.csv")

data

Season  No. in series                          Company Deal  \
0       1.0            1.0                 Ava the Elephant  Yes   
1       1.0            1.0            Mr. Tod's Pie Factory  Yes   
2       1.0            1.0                          Wispots   No   
3       1.0            1.0      College Foxes Packing Boxes   No   
4       1.0            1.0                        Ionic Ear   No   
5       1.0            2.0                   A Perfect Pear  Yes   
6       1.0            2.0                   Classroom Jams  Yes   
7       1.0            2.0                         Lifebelt   No   
8       1.0            2.0                      Crooked Jaw   No   
9       1.0            2.0               Sticky Note Holder   No   
10      1.0            3.0                      Turbobaster  Yes   
11      1.0            3.0                 Stress Free Kids  Yes   
12      1.0            3.0             Kwyzta Chopstick Art   No   
13      1.0            3.0  50 State Capitals in 50 Minutes   No   
14      1.0            3.0                Voyage Air Guitar   No   
15      1.0            4.0                 Gift Card Rescue  Yes   
16      1.0            4.0                        Coverplay  Yes   
17      1.0            4.0        Graffiti Removal Services   No   
18      1.0            4.0              Souls Calling, Inc.   No   
19      1.0            4.0               Coffee Brand Gifts   No   
20      1.0            5.0               My Therapy Journal  Yes   
21      1.0            5.0                         Body Jac  Yes   
22      1.0            5.0          Good Grief Celebrations   No   
23      1.0            5.0                  Granola Gourmet   No   
24      1.0            5.0                        Face Blok   No   
25      1.0            6.0            Pork Barrel BBQ Sauce  Yes   
26      1.0            6.0                     Element Bars  Yes   
27      1.0            6.0              Underease Underwear   No   
28      1.0            6.0               Kalyx Technologies   No   
29      1.0            6.0                         The Fizz   No   
..      ...            ...                              ...  ...   
465     6.0           22.0                      acqua vault  Yes   
466     6.0           22.0                             Naja   No   
467     6.0           23.0               Bee Sweet Lemonade  Yes   
468     6.0           23.0                   Brand Yourself   No   
469     6.0           23.0                         iC Pooch   No   
470     6.0           23.0                       The Home T   No   
471     6.0           24.0                          Budsies   No   
472     6.0           24.0                     bee thinking   No   
473     6.0           24.0                        PullyPalz  Yes   
474     6.0           24.0                            Forus   No   
475     6.0           25.0                       NeatCheeks  Yes   
476     6.0           25.0                            Melni  Yes   
477     6.0           25.0                  Beneath the Ink   No   
478     6.0           25.0                         PittMoss  Yes   
479     6.0           26.0                          ZinePak  Yes   
480     6.0           26.0                       snagastool   No   
481     6.0           26.0                       Buck Mason   No   
482     6.0           26.0                            Noene  Yes   
483     6.0           27.0                           AirCar  Yes   
484     6.0           27.0                 Paleo Diet Foods   No   
485     6.0           27.0     World Record Striper Company  Yes   
486     6.0           27.0                          Frameri   No   
487     6.0           28.0                             Zoom  Yes   
488     6.0           28.0                   Sunscreen Mist   No   
489     6.0           28.0                    SynDaver Labs  Yes   
490     6.0           28.0                     You Kick Ass  Yes   
491     6.0           29.0                      Shark Wheel  

There is one column for each of the "sharks" (a.k.a. investors). A 1 indicates that they chose to invest in that company, while a missing value indicates that they did not choose to invest in that company. Notice that these missing values show up as NaNs when we read in the data. Learn about Pandas' [missing data handling](http://pandas.pydata.org/pandas-docs/stable/missing_data.html), and use these methods to replace all of the missing values with zeros.

In [2]:
# YOUR CODE HERE
#raise NotImplementedError()
#see tip on 04-07 announcements


data.ix[:, 'Corcoran':'Harrington'] = data.ix[:, 'Corcoran':'Harrington'].fillna(0)

"""
data['Corcoran'] = data['Corcoran'].fillna(0)
data['Cuban'] = data['Cuban'].fillna(0)
data['Greiner'] = data['Greiner'].fillna(0)
data['Herjavec'] = data['Herjavec'].fillna(0)
data['John'] = data['John'].fillna(0)
data['O\'Leary'] = data['O\'Leary'].fillna(0)
data['Harrington'] = data['Harrington'].fillna(0)
#data['Guest'] = data['Guest'].fillna(0)
#not including Guest
"""

data

Season  No. in series                          Company Deal  \
0       1.0            1.0                 Ava the Elephant  Yes   
1       1.0            1.0            Mr. Tod's Pie Factory  Yes   
2       1.0            1.0                          Wispots   No   
3       1.0            1.0      College Foxes Packing Boxes   No   
4       1.0            1.0                        Ionic Ear   No   
5       1.0            2.0                   A Perfect Pear  Yes   
6       1.0            2.0                   Classroom Jams  Yes   
7       1.0            2.0                         Lifebelt   No   
8       1.0            2.0                      Crooked Jaw   No   
9       1.0            2.0               Sticky Note Holder   No   
10      1.0            3.0                      Turbobaster  Yes   
11      1.0            3.0                 Stress Free Kids  Yes   
12      1.0            3.0             Kwyzta Chopstick Art   No   
13      1.0            3.0  50 State Capitals in 50 Minutes   No   
14      1.0            3.0                Voyage Air Guitar   No   
15      1.0            4.0                 Gift Card Rescue  Yes   
16      1.0            4.0                        Coverplay  Yes   
17      1.0            4.0        Graffiti Removal Services   No   
18      1.0            4.0              Souls Calling, Inc.   No   
19      1.0            4.0               Coffee Brand Gifts   No   
20      1.0            5.0               My Therapy Journal  Yes   
21      1.0            5.0                         Body Jac  Yes   
22      1.0            5.0          Good Grief Celebrations   No   
23      1.0            5.0                  Granola Gourmet   No   
24      1.0            5.0                        Face Blok   No   
25      1.0            6.0            Pork Barrel BBQ Sauce  Yes   
26      1.0            6.0                     Element Bars  Yes   
27      1.0            6.0              Underease Underwear   No   
28      1.0            6.0               Kalyx Technologies   No   
29      1.0            6.0                         The Fizz   No   
..      ...            ...                              ...  ...   
465     6.0           22.0                      acqua vault  Yes   
466     6.0           22.0                             Naja   No   
467     6.0           23.0               Bee Sweet Lemonade  Yes   
468     6.0           23.0                   Brand Yourself   No   
469     6.0           23.0                         iC Pooch   No   
470     6.0           23.0                       The Home T   No   
471     6.0           24.0                          Budsies   No   
472     6.0           24.0                     bee thinking   No   
473     6.0           24.0                        PullyPalz  Yes   
474     6.0           24.0                            Forus   No   
475     6.0           25.0                       NeatCheeks  Yes   
476     6.0           25.0                            Melni  Yes   
477     6.0           25.0                  Beneath the Ink   No   
478     6.0           25.0                         PittMoss  Yes   
479     6.0           26.0                          ZinePak  Yes   
480     6.0           26.0                       snagastool   No   
481     6.0           26.0                       Buck Mason   No   
482     6.0           26.0                            Noene  Yes   
483     6.0           27.0                           AirCar  Yes   
484     6.0           27.0                 Paleo Diet Foods   No   
485     6.0           27.0     World Record Striper Company  Yes   
486     6.0           27.0                          Frameri   No   
487     6.0           28.0                             Zoom  Yes   
488     6.0           28.0                   Sunscreen Mist   No   
489     6.0           28.0                    SynDaver Labs  Yes   
490     6.0           28.0                     You Kick Ass  Yes   
491     6.0           29.0                      Shark Wheel  

Notice that Amount and Equity are currently being treated as categorical variables (dtype: object). This is because they contain non-numerical characters, like $ and %, so Pandas assumes they are strings. Use [Pandas' string processing capabilities](http://pandas.pydata.org/pandas-docs/stable/text.html) to strip these characters. Then, convert these columns to quantitative variables (i.e., with a dtype of int or float).

In [3]:
# YOUR CODE HERE
#raise NotImplementedError()
"""
#only strips from begin or end...
data.Amount = data.Amount.str.strip(',') # doesn't work

"""
data.Amount = data.Amount.str.replace(',', '')

data.Amount = data.Amount.str.lstrip('$')
data.Equity = data.Equity.str.rstrip('%')

#fill NaN's first...
data['Amount'] = data['Amount'].fillna(0)
data['Equity'] = data['Equity'].fillna(0)

data['Amount'] = data['Amount'].replace('n/a', '')
data['Equity'] = data['Equity'].replace('n/a', '')

#convert categorical var 'Amount' and 'Equity' to a quantitative var; 
#currently dtype of Amount and Equity is Object, check with data.dtypes


data.Amount = pd.to_numeric(data.Amount)
#data.dtypes

#data.Equity = data.Equity.astype(float)
#Strangely, Strings in Equity will not convert to float... so alternatively,

data.Equity = pd.to_numeric(data.Equity)
#data.dtypes


data



Season  No. in series                          Company Deal  \
0       1.0            1.0                 Ava the Elephant  Yes   
1       1.0            1.0            Mr. Tod's Pie Factory  Yes   
2       1.0            1.0                          Wispots   No   
3       1.0            1.0      College Foxes Packing Boxes   No   
4       1.0            1.0                        Ionic Ear   No   
5       1.0            2.0                   A Perfect Pear  Yes   
6       1.0            2.0                   Classroom Jams  Yes   
7       1.0            2.0                         Lifebelt   No   
8       1.0            2.0                      Crooked Jaw   No   
9       1.0            2.0               Sticky Note Holder   No   
10      1.0            3.0                      Turbobaster  Yes   
11      1.0            3.0                 Stress Free Kids  Yes   
12      1.0            3.0             Kwyzta Chopstick Art   No   
13      1.0            3.0  50 State Capitals in 50 Minutes   No   
14      1.0            3.0                Voyage Air Guitar   No   
15      1.0            4.0                 Gift Card Rescue  Yes   
16      1.0            4.0                        Coverplay  Yes   
17      1.0            4.0        Graffiti Removal Services   No   
18      1.0            4.0              Souls Calling, Inc.   No   
19      1.0            4.0               Coffee Brand Gifts   No   
20      1.0            5.0               My Therapy Journal  Yes   
21      1.0            5.0                         Body Jac  Yes   
22      1.0            5.0          Good Grief Celebrations   No   
23      1.0            5.0                  Granola Gourmet   No   
24      1.0            5.0                        Face Blok   No   
25      1.0            6.0            Pork Barrel BBQ Sauce  Yes   
26      1.0            6.0                     Element Bars  Yes   
27      1.0            6.0              Underease Underwear   No   
28      1.0            6.0               Kalyx Technologies   No   
29      1.0            6.0                         The Fizz   No   
..      ...            ...                              ...  ...   
465     6.0           22.0                      acqua vault  Yes   
466     6.0           22.0                             Naja   No   
467     6.0           23.0               Bee Sweet Lemonade  Yes   
468     6.0           23.0                   Brand Yourself   No   
469     6.0           23.0                         iC Pooch   No   
470     6.0           23.0                       The Home T   No   
471     6.0           24.0                          Budsies   No   
472     6.0           24.0                     bee thinking   No   
473     6.0           24.0                        PullyPalz  Yes   
474     6.0           24.0                            Forus   No   
475     6.0           25.0                       NeatCheeks  Yes   
476     6.0           25.0                            Melni  Yes   
477     6.0           25.0                  Beneath the Ink   No   
478     6.0           25.0                         PittMoss  Yes   
479     6.0           26.0                          ZinePak  Yes   
480     6.0           26.0                       snagastool   No   
481     6.0           26.0                       Buck Mason   No   
482     6.0           26.0                            Noene  Yes   
483     6.0           27.0                           AirCar  Yes   
484     6.0           27.0                 Paleo Diet Foods   No   
485     6.0           27.0     World Record Striper Company  Yes   
486     6.0           27.0                          Frameri   No   
487     6.0           28.0                             Zoom  Yes   
488     6.0           28.0                   Sunscreen Mist   No   
489     6.0           28.0                    SynDaver Labs  Yes   
490     6.0           28.0                     You Kick Ass  Yes   
491     6.0           29.0                      Shark Wheel  

The valuation of a company is how much it is worth. If someone invests \$10,000 for a 40% equity stake in the company, then this means the company must be valued at \$25,000, since 40% of \$25,000 is \$10,000.

Calculate the valuation of each funded company. Which company was most valuable? Is it the same as the company that received the largest investment from the sharks? (First write your code in the code cell below, then write your text answer in the markdown cell below.)

In [4]:
# YOUR CODE HERE
#raise NotImplementedError()

data['Company Valuation'] = data['Amount']/ (data['Equity']/100)
#data['Company Valuation'].max()

"""data.ix[data[ ['Company Valuation'] ] == data[ ['Company Valuation'] ].max(None, None, 
  None, float)]"""

#data.ix[data['Company Valuation'] == data['Company Valuation'].max() ]
#CONDITIONED on Company Valuation is the largest. It comes from conditional probability.
# returned the company with INIFINITE valuation


#Instead, sort Company Valuations in descending order
data.sort_values(by='Company Valuation', ascending=False)


Season  No. in series                          Company Deal  \
312     5.0           13.0                 The Wall DoctoRX  Yes   
421     6.0           11.0                             Zipz  Yes   
464     6.0           22.0                   Emazing Lights  Yes   
489     6.0           28.0                    SynDaver Labs  Yes   
483     6.0           27.0                           AirCar  Yes   
284     5.0            6.0  Ten Thirty One Haunted Hayrides  Yes   
384     6.0            2.0                        Roominate  Yes   
347     5.0           22.0                           Plated  Yes   
363     5.0           26.0                    Rugged Maniac  Yes   
395     6.0            5.0           The Red Dress Boutique  Yes   
457     6.0           20.0                        Keen Home  Yes   
220     4.0           16.0                  HDYR Sushi Bars  Yes   
476     6.0           25.0                            Melni  Yes   
479     6.0           26.0                          ZinePak  Yes   
261     4.0           26.0                       Tom n Chee  Yes   
264     5.0            1.0              Postcard on the Run  Yes   
133     3.0            9.0                       Miso Media  Yes   
412     6.0            9.0                    Squatty Potty  Yes   
436     6.0           15.0                            Lumio  Yes   
280     5.0            5.0      Ruffle Butts - Rugged Butts  Yes   
268     5.0            2.0                     Breathometer  Yes   
367     5.0           27.0                        Angellift  Yes   
324     5.0           16.0                       Cycloramic  Yes   
400     6.0            6.0                BeatBox Beverages  Yes   
440     6.0           16.0                        Phonesoap  Yes   
336     5.0           19.0                       RevoLights  Yes   
439     6.0           16.0                         GreenBox  Yes   
491     6.0           29.0                      Shark Wheel  Yes   
69      2.0            2.0       First Defense Nasal Screen  Yes   
494     6.0           29.0                        Spikeball  Yes   
..      ...            ...                              ...  ...   
426     6.0           12.0                         Eve Drop   No   
429     6.0           13.0                      SkinnyShirt   No   
430     6.0           13.0               Coffee Meets Bagel   No   
434     6.0           14.0                         Vestpakz   No   
437     6.0           15.0                      Bello Verde   No   
438     6.0           15.0                          Napwell   No   
441     6.0           16.0               Victoria's Kitchen   No   
442     6.0           16.0               Tycoon Real Estate   No   
445     6.0           17.0            Balm Chicky Balm Balm   No   
446     6.0           17.0                           BedJet   No   
449     6.0           18.0                      The Lip Bar   No   
450     6.0           18.0               Himalayan Dog Chew   No   
453     6.0           19.0                    Sseko Designs   No   
454     6.0           19.0                             Lumi   No   
458     6.0           20.0                          Taaluma   No   
460     6.0           21.0                         BedRyder   No   
466     6.0           22.0                             Naja   No   
468     6.0           23.0                   Brand Yourself   No   
469     6.0           23.0                         iC Pooch   No   
470     6.0           23.0                       The Home T   No   
471     6.0           24.0                          Budsies   No   
472     6.0           24.0                     bee thinking   No   
474     6.0           24.0                            Forus   No   
477     6.0           25.0                  Beneath the Ink   No   
480     6.0           26.0                       snagastool   No   
481     6.0           26.0                       Buck Mason   No   
484     6.0           27.0                 Paleo Diet Foods  

In [5]:
data.ix[data['Amount'] == data['Amount'].max() ]

Season  No. in series Company Deal         Industry Entrepreneur Gender  \
483     6.0           27.0  AirCar  Yes  Green/CleanTech                Male   

      Amount  Equity  Corcoran  Cuban  Greiner  Herjavec  John  O'Leary  \
483  5000000    50.0       0.0    0.0      0.0       1.0   0.0      0.0   

     Harrington  Guest                                    Details / Notes  \
483         0.0    NaN  Contingent on getting deal to bring to contine...   

     Company Valuation  
483         10000000.0

Although "The Wall DoctoRX" is technically the most valuable company, it is infinite, so I excluded it and instead concluded that "Zipz" was the most valuable company. This is not the same company as "AirCar", which received the most investment from the sharks.

Calculate the total amount of money that each shark invested over the 6 seasons. (Note: If $n$ sharks funded a given venture, then the amount that each shark invested is the amount divided by $n$.)

In [6]:
# YOUR CODE HERE
#raise NotImplementedError()
data['Guest'] = data['Guest'].fillna(0)

data[ 'Number of Sharks That Invested in Company (n)'] = data.ix[:,'Corcoran':'Guest'].sum(axis = 1)
data['Total Amount Each Shark Invested'] = \
data['Amount']/data[ 'Number of Sharks That Invested in Company (n)']
# '\' will let me write wrap code to next line

Corcoran_total_invested = data.ix[data['Corcoran'] == 1, 'Total Amount Each Shark Invested'].sum()
# only take rows where corcoran invested and add total amount each shark invested to sum and save to var

print(Corcoran_total_invested)

Cuban_total_invested = data.ix[data['Cuban'] == 1, 'Total Amount Each Shark Invested'].sum()

print(Cuban_total_invested)

Greiner_total_invested = data.ix[data['Greiner'] == 1, 'Total Amount Each Shark Invested'].sum()

print(Greiner_total_invested)

Herjavec_total_invested = data.ix[data['Herjavec'] == 1, 'Total Amount Each Shark Invested'].sum()

print(Herjavec_total_invested)

John_total_invested = data.ix[data['John'] == 1, 'Total Amount Each Shark Invested'].sum()

print(John_total_invested)

OLeary_total_invested = data.ix[data['O\'Leary'] == 1, 'Total Amount Each Shark Invested'].sum()

print(OLeary_total_invested)

Harrington_total_invested = data.ix[data['Harrington'] == 1, 'Total Amount Each Shark Invested'].sum()

print(Harrington_total_invested)

Guest_total_invested = data.ix[data['Guest'] == 1, 'Total Amount Each Shark Invested'].sum()

print(Guest_total_invested)

4912500.0
17817500.0
8170000.0
16297500.0
8154000.0
7952500.0
800000.0
400000.0


Corcoran: 4912500.0

Cuban: 17817500.0

Greiner: 8170000.0

Herjavec: 16297500.0

John: 8154000.0

O'Leary: 7952500.0

Harrington: 800000.0

Guest: 400000.0

What percentage of companies led by male entrepreneurs were funded? What percentage of companies led by female entrepreneurs were funded? Report 95% confidence intervals for each.

In [7]:
# YOUR CODE HERE
#raise NotImplementedError()
from scipy.stats import norm
import numpy as np



m_total = data.ix[ (data["Entrepreneur Gender"] == "Male")  , "Deal" ].count()
f_total = data.ix[ (data["Entrepreneur Gender"] == "Female")  , "Deal" ].count()



"""
print(m_total)
print(f_total)
"""

m_funded = data.ix[(data["Entrepreneur Gender"]  == 'Male') & (data["Deal"] == 'Yes') , "Deal"].count()

f_funded = data.ix[(data["Entrepreneur Gender"]  == 'Female') & (data["Deal"] == 'Yes') , "Deal"].count()

"""
print(m_funded)
print(f_funded)
"""

m_funded_percentage = (m_funded/m_total) 
f_funded_percentage = (f_funded/f_total)

print(m_funded_percentage)
print(f_funded_percentage)


m_lower_bound = m_funded_percentage - norm.ppf(.975) * \
    np.sqrt( (m_funded_percentage - m_funded_percentage**2) /m_total)

m_upper_bound = m_funded_percentage + norm.ppf(.975) * \
    np.sqrt( (m_funded_percentage - m_funded_percentage**2 )/m_total) 

f_lower_bound = f_funded_percentage - norm.ppf(.975) * \
    np.sqrt( (f_funded_percentage - f_funded_percentage**2) /f_total) 

f_upper_bound = f_funded_percentage + norm.ppf(.975) *\
    np.sqrt( (f_funded_percentage - f_funded_percentage**2) /f_total)  

print(m_lower_bound)
print(m_upper_bound)
print(f_lower_bound)
print(f_upper_bound)


0.481355932203
0.536
0.424338854499
0.538373009908
0.448575235795
0.623424764205


percentage of companies led by male entrepreneurs that were funded: 48.14%

percentage of companies led by female entrepreneurs that were funded: 53.6%

Percentage of companies led by Males that were funded 95% CI: (42.4338854499,53.8373009908)

Percentage of companies led by Females that were funded 95% CI: (44.8575235795,62.3424764205)

Find something else interesting in this data set. (Note: This question is deliberately open-ended. Extra credit is available if you come up with something really neat!)

In [8]:
# YOUR CODE HERE
#raise NotImplementedError()

def range(x):
    return max(x) - min(x)

data["Total Amount Each Shark Invested"] = data["Total Amount Each Shark Invested"].fillna(0)
data["Company Valuation"] = data["Company Valuation"].fillna(0)
data.groupby(["Industry"]).agg(range)

Season  No. in series   Amount  Equity  Corcoran  \
Industry                                                                  
Business Services         4.0           25.0   250000    50.0       0.0   
Children / Education      5.0           28.0   500000    51.0       1.0   
Consumer Products         5.0           24.0   500000    70.0       1.0   
Fashion / Beauty          5.0           26.0  1200000   100.0       1.0   
Fitness / Sports          5.0           28.0  1750000    50.0       1.0   
Food and Beverage         5.0           28.0  2500000   100.0       1.0   
Green/CleanTech           5.0           22.0  5000000    50.0       0.0   
Healthcare                5.0           27.0  3000000    55.0       1.0   
Lifestyle / Home          5.0           28.0   750000   100.0       1.0   
Media / Entertainment     5.0           25.0  2000000    30.0       0.0   
Pet Products              5.0           24.0   150000    60.0       1.0   
Software / Tech           5.0           28.0   500000    50.0       1.0   
Uncertain / Other         5.0           28.0  1250000   100.0       0.0   

                       Cuban  Greiner  Herjavec  John  O'Leary  Harrington  \
Industry                                                                     
Business Services        1.0      0.0       1.0   0.0      1.0         0.0   
Children / Education     1.0      1.0       1.0   1.0      1.0         0.0   
Consumer Products        1.0      1.0       1.0   1.0      0.0         0.0   
Fashion / Beauty         1.0      1.0       1.0   1.0      1.0         1.0   
Fitness / Sports         1.0      0.0       1.0   1.0      1.0         1.0   
Food and Beverage        1.0      1.0       1.0   1.0      1.0         1.0   
Green/CleanTech          1.0      1.0       1.0   0.0      1.0         0.0   
Healthcare               1.0      1.0       1.0   1.0      1.0         0.0   
Lifestyle / Home         1.0      1.0       1.0   1.0      1.0         0.0   
Media / Entertainment    1.0      1.0       1.0   1.0      0.0         0.0   
Pet Products             1.0      1.0       0.0   1.0      0.0         1.0   
Software / Tech          1.0      1.0       1.0   1.0      1.0         0.0   
Uncertain / Other        1.0      0.0       1.0   0.0      1.0         0.0   

                       Guest  Company Valuation  \
Industry                                          
Business Services        0.0       1.250000e+06   
Children / Education     0.0       1.000000e+07   
Consumer Products        0.0       2.000000e+06   
Fashion / Beauty         0.0       6.000000e+06   
Fitness / Sports         1.0       7.000000e+06   
Food and Beverage        1.0       2.500000e+07   
Green/CleanTech          1.0       1.000000e+07   
Healthcare               0.0       1.200000e+07   
Lifestyle / Home         0.0                inf   
Media / Entertainment    0.0       1.300000e+07   
Pet Products             0.0       7.500000e+05   
Software / Tech          0.0       3.750000e+06   
Uncertain / Other        1.0       4.166667e+06   

                       Number of Sharks That Invested in Company (n)  \
Industry                                                               
Business Services                                                3.0   
Children / Education                                             5.0   
Consumer Products                                                2.0   
Fashion / Beauty                                                 3.0   
Fitness / Sports                                                 3.0   
Food and Beverage                                                4.0   
Green/CleanTech                                                  3.0   
Healthcare                                                       5.0   
Lifestyle / Home                                                 5.0   
Media / Entertainment                                            2.0   
Pet Products                                                     2.0   
Software / Tech         

Based on the ranges of values under each variable calculated for each different industry, an industry that every shark, including the Guest, thought was important at one time was Food/Beverage (a range of 1.0 for an industry means that a shark has decided to invest in at least one company that belonged to it over the 6 seasons). Based on Cuban's decision to invest at least once in every industry, it showed that each industry shown on the table has some importance. In other words, he was the only shark that did not demonstrate any strong lack of interest in any industry.